misaligned.cpp

```diff
@@ -8,14 +14,76 @@ int printColorMap() {
     for(i = 0; i < 5; i++) {
         for(j = 0; j < 5; j++) {
             std::cout << i * 5 + j << " | " << majorColor[i] << " | " << minorColor[i] << "\n";
+            colorPair.insert(std::make_pair(std::make_pair(majorColor[i], minorColor[j]),(i * 5 + j)));
         }
     }
     return i * j;
 }
 
+int printColorMapManual() {
+    const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
+    const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+    int maxWdth = 6;
+    int i = 0, j = 0;
+    for(i = 0; i < 5; i++) {
+        for(j = 0; j < 5; j++) {
+            if(i * 5 + j+1 < 10)

```

Comment:
The number 5 used in the pair number calculation would lead to confusion or error in future when someone else is maintaining the code.
Also, will this code work if major color or minor color length is other than 5. Please think over it and write test cases accordingly to cover such cases.